mmv1/templates/terraform/encoders/vertex_ai_models.go.tmpl

```diff
@@ -0,0 +1,6 @@
+if modelName, ok := obj["modelName"].(string); ok && modelName != "" {
+	obj["displayName"] = modelName
+	delete(obj, "modelName")
+}
```

Comment:
Is this required given that this PR has displayName in the schema (https://github.com/GoogleCloudPlatform/magic-modules/commit/f1124737c6c9bba113958949f79f51ee608a90d0)?

Reply:
no this would be implemented for when we include `:UPLOAD`. removing right now.